src/config/config.js

```diff
@@ -4,7 +4,7 @@ require('dotenv').config()
 
 module.exports = {
 
-  port: process.env.PORT || 8000,
+  port: process.env.WASTE_PERMITS_APP_PORT || 8000,
```

Comment:
As above, it needs to remain as `PORT`.

Reply:
This didn't work for me because as PORT because I had WASTE_PERMITS_APP_PORT in my .env file. I hadn't realised it had changed back to PORT again, I thought it was a bug in the code.

I have updated my .env file and changed the code back and it seems to be working OK now.